test/aqua/operators/test_pauli_expectation.py

```diff
@@ -89,7 +89,8 @@ def test_pauli_expect_op_vector(self):
         # !!NOTE!!: Depolarizing channel (Sampling) means interference
         # does not happen between circuits in sum, so expectation does
         # not equal expectation for Zero!!
-        np.testing.assert_array_almost_equal(sampled_zero_mean.eval(), [0, 0, 0, 2], decimal=1)
+        np.testing.assert_array_almost_equal(sampled_zero_mean.eval(), [0, 0, 0, 2 ** 0.5],
```

Comment:
Is there a reason why the outcome here has changed, the last digit from 2 to square root of 2?

Reply:
Glad you asked! This was a bug, before the global coefficient has been ignored, such that
```
ops = ListOp([X, Y, Z, I])
state = 0 * (Plus + Minus)
sampler = CircuitSampler(Aer.get_backend('statevector_simulator')).convert(~StateFn(ops) @ state)
sampler.eval()  # returns [0, 0, 0, 2] !!
```
now that is fixed and the above returns `[0, 0, 0, 0]`. Since the global coefficient is now included we get a factor of 1/sqrt(2), hence sqrt(2) instead of 2.

Reply:
Thinking about this more, there is some deeper concept we have to discuss at some point. Evaluating this expectation value with one of the expectation converters should give the correct result of `[0, 0, 1, 1]` I think. But that's another issue.

Reply:
Moved this to #1077